src/main/java/com/denizenscript/denizencore/scripts/commands/core/MongoCommand.java

```diff
@@ -0,0 +1,521 @@
+    //
+    // The mongo command is merely a wrapper, and further usage details should be gathered from an official MongoDB command reference rather than from Denizen command help.
+    // You can view the official redis documentation and the supported commands here: <@link url https://www.mongodb.com/docs/manual/introduction/>
+    //
+    // @Tags
+    // <util.mongo_connections> returns a ListTag of all the current Mongo connections.
+    // <entry[saveName].result> returns the text result sent back from Mongo in a JSON format. JSON can be in an ElementTag or a ListTag depending on the action run.
+    // <entry[saveName].inserted_id> returns the ID of the item that has been inserted via the `insert` action.
+    // <entry[saveName].ok> returns the 'ok' value from the result. Used with the `command` action.
+    // <entry[saveName].upserted_id> returns the ID the upserted item. Returned if the `upsert` bool is true when updating.
+    // <entry[saveName].updated_count> returns the amount of Documents updated via the `update` action.
+    //
+    // @Usage
+    // Use to connect to a Mongo instance.
+    // - ~mongo id:name connect:<secret[uri]>
+    //
+    // @Usage
+    // Use to disconnect from a Mongo instance.
+    // - mongo id:name disconnect
+    //
+    // @Usage
+    // Run a simple command.
+    // - ~mongo id:name command:<map[dbStats=1]>
+    //
+    // @Usage
+    // Run more complex commands.
+    // - definemap commands:
+    //      count: my_collection
+    //      skip: 4
+    //  - ~mongo id:name command:<[commands]>
+    //
+    // @Usage
+    // Simple find query.
+    // - ~mongo id:name find:<map[name=Bob]>
+    //
+    // @Usage
+    // Complex find query with query filters.
+    // - definemap filters:
+    //      $and:
+    //          - number_greater_than:
+    //              $gt: 2
+    //          - number_less_than:
+    //              $lt: 5
+    // - ~mongo id:name find:<[filters]>
+    //
+    // @Usage
+    // Insert data into a Collection.
+    // - definemap data:
+    //      name: Pluto
+    //      order_from_sun: 9
+    //      has_rings: false
+    //      main_atmosphere:
+    //          - N2
+    //          - CH4
+    //          - CO
+    // - ~mongo id:name insert:<[data]> save:mg
+    //
+    // @Usage
+    // Update data.
+    // - definemap old_data:
+    //      name: Pluto
+    // - definemap new_data:
+    //      $set:
+    //          name: Pluto (A Dwarf Planet)
+    // - ~mongo id:name update:<[old_data]> new:<[new_data]>
+    //
+    // @Usage
+    // Change Databases.
+    // - ~mongo id:name use_database:my_new_database
+    //
+    // @Usage
+    // Change Collections.
+    // - ~mongo id:name use_collection:my_new_collection
+    // -->
+
+    public static Map<String, MongoClient> connections = new HashMap<>();
+    public static Map<String, MongoDatabase> databases = new HashMap<>();
+    public static Map<String, MongoCollection<Document>> collections = new HashMap<>();

```

Comment:
Rather than triple maps, there should be a single map and a sub-class containing all 3 values